test/util/util.go

```diff
@@ -967,8 +967,40 @@ func KExecute(ctx context.Context, cfg *rest.Config, client *rest.RESTClient, ns
 	return out.Bytes(), outErr.Bytes(), nil
 }
 
+// GetProjectBaseDir retrieves the project base directory either from an environment variable or by searching for a Makefile.
```

Comment:
```suggestion
// GetProjectBaseDir retrieves the project base directory either from an environment variable or by searching for a Makefile. The fallback to the search is useful for running in IDEs like vs-code which don't set the PROJECT_DIR
env. variable by default.
```
I'm fine with that comment for tests, where the audience is other contributors.

Reply:
Yes, that was the purpose, so they know the intention behind it

Reply:
added